advocacy_docs/partner_docs/PreciselyConnectCDC/05-Usingpreciselyconnectcdc.mdx

```diff
@@ -0,0 +1,321 @@
+---
+title: 'Using'
+description: 'Walkthrough of example usage scenarios'
+---
+
+The following steps demonstrate Precisely Connect CDC’s Replication functionality with EDB Postgres Advanced Server.
+
+## Configure EDB Postgres Advanced Server as Source Instance in Connect CDC Director
```

Comment:
## Configure EDB Postgres Distribution as a Source Instance in Connect CDC Director

Reply:
Guide updated.